Sources/Onboarding/Welcome/WelcomeViewCallout.swift

```diff
@@ -124,8 +124,15 @@ class WelcomeViewCallout: UIView {
   }
 
   private let primaryButton = RoundInterfaceButton(type: .custom).then {
-    $0.setTitleColor(.white, for: .normal)
-    $0.backgroundColor = .braveBlurpleTint
+    $0.configuration = .filled()
+    $0.configuration?.showsActivityIndicator = false
+    $0.configuration?.imagePadding = 5
+    $0.configuration?.activityIndicatorColorTransformer = UIConfigurationColorTransformer({ _ in
+      .white
+    })
+    $0.configuration?.baseForegroundColor = .white
+    $0.configuration?.baseBackgroundColor = .braveBlurpleTint
```

Comment:
Why do we change it in this PR?

Reply:
Here problem was with iOS 17, adding subview and updating properties over subviews inside UIButton is not working. So When an indicator is added inside button and startAnimate is called it will not update its state UI etc

A issue is created related with this https://github.com/brave/brave-ios/issues/8555

To overcome this issue button configuration is used and for this to work properly with loading it should have base Foreground color and base Background color.